From 17b605b420cf98528f53bcb26344f7875070d82d Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Tue, 28 Mar 2023 16:43:12 -0700
Subject: [PATCH] Add Android platforms

---
 Makefile.libretro | 17 +++++++++++++++++
 1 file changed, 17 insertions(+)

diff --git a/Makefile.libretro b/Makefile.libretro
index 4b0cdb6..2e50dfa 100644
--- a/Makefile.libretro
+++ b/Makefile.libretro
@@ -29,6 +29,23 @@ ifneq (,$(findstring unix,$(platform)))
 		AARCH64 = 1
 		MORE_CFLAGS += -DCPU_AARCH64
 	endif
+else ifneq (,$(findstring armeabi-v7a,$(platform)))
+	TARGET := $(TARGET_NAME)_libretro.so
+	fpic := -fPIC
+	LDFLAGS := -lz -lpthread -lFLAC -lmpg123 -ldl
+	SHARED := -shared -Wl,--version-script=$(CORE_DIR)/libretro/link.T 
+	CFLAGS += $(PLATFORM_DEFINES)
+	MORE_CFLAGS += -DARMV6T2 -DARM_HAS_DIV
+	HAVE_NEON = 1
+	USE_PICASSO96 = 1
+	CPU_FLAGS +=  -marm
+else ifneq (,$(findstring arm64-v8a,$(platform)))
+	TARGET := $(TARGET_NAME)_libretro.so
+	fpic := -fPIC
+	LDFLAGS := -lz -lpthread -lFLAC -lmpg123 -ldl
+	SHARED := -shared -Wl,--version-script=$(CORE_DIR)/libretro/link.T 
+	AARCH64 = 1
+	MORE_CFLAGS += -DCPU_AARCH64
 else ifneq (,$(findstring rpi4,$(platform)))
 	TARGET := $(TARGET_NAME)_libretro.so
 	fpic = -fPIC
-- 
2.34.1

